src/configuration/js-team/index.js

```diff
@@ -0,0 +1,35 @@
+import Linux64 from './linux64';
+import MacConfig from './macOS';
+import Win7Config from './win7';
+import Win10Config from './win10';
+import AndroidConfig from './android';
+
+const platforms = {
```

Comment:
Renamed it from `platforms` to `views`. That will help us detach from the concept of `platforms` and emphasize more that this object defines the views.